### PR TITLE
Add SysmonSearch

### PR DESCRIPTION
**Pull request by hslatman**

### CODE REVIEW

**Comment by meirwah** [comment]
@hslatman can you fix conflicts?

**Comment by meirwah** [comment]
@hslatman ?

**Comment by hslatman** [comment]
Just fixed it, @meirwah. I'm looking into fixing notifications (not just from this repository) now.

File: README.md [hunk 1]
```diff
@@ -121,9 +121,10 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 ### Log Analysis Tools
 
-* [Lorg](https://github.com/jensvoid/lorg) - Tool for advanced HTTPD logfile security analysis and forensics.
-* [Logdissect](https://github.com/dogoncouch/logdissect) - CLI utility and Python API for analyzing log files and other data.
+* [Lorg](https://github.com/jensvoid/lorg) - a tool for advanced HTTPD logfile security analysis and forensics
```
Comment (meirwah) [inline README.md]: why this change?

File: README.md [hunk 2]
```diff
@@ -121,9 +121,10 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
...
+* [Logdissect](https://github.com/dogoncouch/logdissect) - A CLI utility and Python API for analyzing log files and other data.
```
Comment (meirwah) [inline README.md]: also this not needed I think

**Comment by hslatman** [comment]
Sorry for that, @meirwah. My edit was crap.
